### PR TITLE
add message that Internet Explorer is not supported

### PR DESCRIPTION
**Pull request by olekstomek**
Related to: https://github.com/bndw/wifi-card/issues/92

Another idea is using [react-device-detect](https://bundlephobia.com/package/react-device-detect@1.17.0).
Example:

```
import {isIE} from 'react-device-detect';

render() {
    if (isIE) return <div> IE is not supported. Download Chrome/Opera/Firefox </div>
    return (
        <div>...content</div>
    )
}
```

I was trying by this way but in IE but in IE, the error is thrown earlier - before it gets to the point where the browser is checked. Because if I did a test that I simulated that Chrome is Internet Explorer (by inverting the if condition) then I got the message correctly.

So I did it via JavaScript. This is just a suggestion

### CODE REVIEW

**Comment by olekstomek** [comment]
Another idea is using [react-device-detect](https://bundlephobia.com/package/react-device-detect@1.17.0).
Example:

```
import {isIE} from 'react-device-detect';

render() {
    if (isIE) return <div> IE is not supported. Download Chrome/Opera/Firefox </div>
    return (
        <div>...content</div>
    )
}
```

I was trying by this way but in IE but in IE, the error is thrown earlier - before it gets to the point where the browser is checked. Because if I did a test that I simulated that Chrome is Internet Explorer (by inverting the if condition) then I got the message correctly.

So I did it via JavaScript. This is just a suggestion

File: public/index.html [hunk 1]
```diff
@@ -39,5 +41,15 @@
...
-  </body>
-</html>
+</body>
+
+<script>
+  // Internet Explorer 6-11
+  const isIE = /*@cc_on!@*/false || !!document.documentMode;
+
+  if (isIE) {
+    document.getElementById("root").innerHTML = "Internet Explorer is not supported. Download Chrome/Opera/Firefox.";
```
Comment (bndw) [inline public/index.html:51]: nit: Please list Firefox first 😄

**Review by bndw — COMMENTED** [review]
I think this is a good idea since there are less-technical folks out there potentially using IE. I have no way to test this, but it's not throwing console errors so I'll go with that.

File: public/index.html (hunk 1 above)
Comment (olekstomek) [reply public/index.html]: Done. :) I added a separate commit with larger font for the message (you can possibly skip this commit if it's unnecessary).
Additional idea for eg adding browser links in this message (users are lazy). What do you think?

Comment (bndw) [reply public/index.html]: I'd prefer to keep the responsibility of this website small -- we'll alert the user but I don't think it's worth going beyond.

**Review by bndw — APPROVED** [review]

**Pull request merged**